src/main/java/org/jabref/gui/preview/PreviewViewer.java

```diff
@@ -93,7 +91,11 @@ public PreviewViewer(BibDatabaseContext database, DialogService dialogService, S
         });
 
     }
-
+    public void setTheme(String theme){
+        if(theme.equals("Dark.css")){
+            previewView.getEngine().setUserStyleSheetLocation(getClass().getResource("Dark_preview.css").toString());
```

Comment:
As you are already loading from an external resource... Why don't you reuse the original `Dark.css` (https://github.com/taerim25/jabref/blob/d1189b51a3853238ae8193589dc92bd7621dbd37/src/main/java/org/jabref/gui/Dark.css) here?